### DISCUSSION TITLE
VS 2022 copilot can not sign in ( v1.4)

### DISCUSSION THREAD

**Discussion by anthonyy41** [community/community#29693] [Copilot Conversations] [answered]
- Visual Studio Community 17.3.0

Today i have update copilot to version v1.4.0
and when i click sign in but not pop any authorize notification
![image](https://user-images.githubusercontent.com/31025028/184909979-db0df927-c6b2-47da-99f6-2cc72b87e86e.png)

I have try reinstall copilot and Visual Studio but still not work
does any suggest or problem ?

Thanks

**Comment by chenhw** [comment]
same problem

**Comment by kzu** [comment, marked as answer]
Plz update to 1.40.0.1 which contains a fix. Thanks for reporting!

**Reply by anthonyy41** [reply]
After update copilot can work 
Thanks for your help!

**Reply by Bortronx** [reply]
After updating copilot it doesn't do suggestions anymore in vs2022 preview
